### PR TITLE
rmtl->pmtl, RMTL->PMTL, Relevant Molecular Target->Pediatric Molecula…

### PR DESCRIPTION
**Pull request by sangeetashukla**
…r Target, v1.0->v1.1

### Purpose/implementation Section
Update `long-format-table-utils` module per recent discussions with FNL

#### What was your approach?

Module has been edited to reflect following changes in occurrence

- rmtl --> pmtl
- RMTL --> PMTL
- Relevant Molecular Target --> Pediatric Molecular Target
- v1.0 --> v1.1

#### What GitHub issue does your pull request address?
[Issue 206](https://github.com/PediatricOpenTargets/ticket-tracker/issues/206)

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
While I have renamed the above occurrences, please clarify if the below change needs to be done.
Non-Relevant Molecular Target  --> Non-Pediatric Molecular Target

#### What types of results are included (e.g., table, figure)?
Renaming is done only in R scripts and documentation in the module.

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

**Comment by logstar** [comment]
@sangeetashukla - Thank you for the updates.

Could you also make the following changes in the following files `annotator/tests/test_data/inspected_annotated_test_long_format_table.tsv` and `annotator/tests/test_data/v7_inspected_annotated_test_long_format_table.tsv`.

(EDIT 09/24/2021 YZ)

- Change "RMTL" to "PMTL" in column names.
- Change "Relevant Molecular Target (RMTL version 1.0)" to ~~"Pediatric Molecular Target (PMTL version 1.1)"~~ "Relevant Molecular Target (PMTL version 1.1)" in "RMTL" column values, according to @zdorman's comment at <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/129#issuecomment-926637311>.

These two files are used for unit testing. You could run unit testing with `bash ./annotator/run-tests.sh`, after copying `ensg-hugo-pmtl-mapping.tsv` to `../../data`. For more details, see `README.md`.

After making the aforementioned changes, all unit tests passed.

```text
$ bash ./annotator/run-tests.sh

✔ |  OK F W S | Context
✔ |  68       | tests/test_annotate_long_format_table.R [24.6 s]                                                                                                        
✔ |  58       | tests/test_annotator_cli.R [64.6 s]                                                                                                                     
✔ |   8       | tests/test_collapse_name_vec.R                                                                                                                          
✔ |   7       | tests/test_collapse_rp_lists.R                                                                                                                          
✔ |  21       | tests/test_helper_import_function.R                                                                                                                     

══ Results ═════════════════════
Duration: 89.4 s

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 162 ]
Done running run-tests.sh
```

**Comment by zdorman** [comment]
Hi @sangeetashukla. Thanks for all the careful updating and tracking, though one change is not as I intended. Values of `Relevant Molecular Target` should not be changed to `Pediatric Molecular Target`. 

In [#206](https://github.com/PediatricOpenTargets/ticket-tracker/issues/206) I included the note: 

> Note that the possible options for designations: `Relevant Molecular Target` or `Non-Relevant Molecular Target` have not changed since v1.0, though the `(RMTL version 1.0)` annotation in the designation should be updated to `(PMTL version 1.1)`.

So the name change only refers to the title of the entire entity (RMTL -> PMTL). Targets within the lists are still either Relevant or Non-Relevant. Sorry if the original request was not clear. Let me know if you'd like more context. 

The other changes appear to be correct:

> rmtl --> pmtl
RMTL --> PMTL
v1.0 --> v1.1

**Review by chinwallaa — APPROVED** [review]
Thanks. Looked thru changes and they look good.

**Comment by sangeetashukla** [comment]
@logstar @zdorman I have made changes as discussed in the above comments.
@chinwallaa Please review the latest changes once more. I am confident they are okay, and with your go-ahead, I will do a final dev merge.

File: analyses/long-format-table-utils/annotator/tests/test_data/inspected_annotated_test_long_format_table.tsv [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+BRAF	ENSG00000157764	Neuroblastoma	GMKF	32.1902392344498	Relevant Molecular Target (PMTL version 1.1)	CosmicCensus,Kinase,Oncogene	Y	Oncogene	B-Raf proto-oncogene, serine/threonine kinase	NP_001341538.1,NP_001361173.1,NP_001361187.1,NP_001365396.1,NP_001365397.1,NP_001365398.1,NP_001365399.1,NP_001365400.1,NP_001365401.1,NP_001365402.1,NP_001365403.1,NP_001365404.1,NP_004324.2	EFO_0000621	MONDO_0005072
+MYCN	ENSG00000134323	Neuroblastoma	all_cohorts	43.3005263157895	Relevant Molecular Target (PMTL version 1.1)	CosmicCensus,Oncogene,TranscriptionFactor	Y	Oncogene	MYCN proto-oncogene, bHLH transcription factor	NP_001280157.1,NP_001280160.1,NP_001280162.1,NP_005369.2	EFO_0000621	MONDO_0005072
+MYCN	ENSG00000134323	Medulloblastoma	all_cohorts	21.2189523809524	Relevant Molecular Target (MTL version 1.1)	CosmicCensus,Oncogene,TranscriptionFactor	Y	Oncogene	MYCN proto-oncogene, bHLH transcription factor	NP_001280157.1,NP_001280160.1,NP_001280162.1,NP_005369.2	EFO_0002939	MONDO_0007959
```
Comment (logstar) [inline analyses/long-format-table-utils/annotator/tests/test_data/inspected_annotated_test_long_format_table.tsv]:
```suggestion
MYCN	ENSG00000134323	Medulloblastoma	all_cohorts	21.2189523809524	Relevant Molecular Target (PMTL version 1.1)	CosmicCensus,Oncogene,TranscriptionFactor	Y	Oncogene	MYCN proto-oncogene, bHLH transcription factor	NP_001280157.1,NP_001280160.1,NP_001280162.1,NP_005369.2	EFO_0002939	MONDO_0007959
```

**Review by logstar — APPROVED** [review]
The latest changes look good to me. One minor suggestion to fix test TSV file typo.

All unit tests passed using the latest `ensg-hugo-pmtl-mapping.tsv` in PR #127 <https://raw.githubusercontent.com/PediatricOpenTargets/OpenPedCan-analysis/f37679d09e4d901fb23444d2ccdfd3547e49ee43/analyses/ensg-hugo-rmtl-mapping/results/ensg-hugo-pmtl-mapping.tsv>.

```text
$ bash annotator/run-tests.sh 
✔ |  OK F W S | Context
✔ |  68       | tests/test_annotate_long_format_table.R [24.6 s]                                                                
✔ |  58       | tests/test_annotator_cli.R [66.0 s]                                                                             
✔ |   8       | tests/test_collapse_name_vec.R                                                                                  
✔ |   7       | tests/test_collapse_rp_lists.R                                                                                  
✔ |  21       | tests/test_helper_import_function.R                                                                             

══ Results ═════════════════════════
Duration: 90.7 s

[ FAIL 0 | WARN 0 | SKIP 0 | PASS 162 ]
Done running run-tests.sh
```

File: analyses/long-format-table-utils/annotator/tests/test_data/inspected_annotated_test_long_format_table.tsv (hunk 1 above)
Comment (sangeetashukla) [reply analyses/long-format-table-utils/annotator/tests/test_data/inspected_annotated_test_long_format_table.tsv]: @logstar I have made this change, and merged to dev.

**Comment by runjin326** [comment]
@sangeetashukla, so this branch now has most up-to-date dev merged and is ready to be merged?

**Comment by sangeetashukla** [comment]
> @sangeetashukla, so this branch now has most up-to-date dev merged and is ready to be merged?

@runjin326 Thank you for merging the branch. I was able to pull the merged changes into my fork.

**Pull request merged**